opcpa_tpr_config/widgets.py

```diff
@@ -153,17 +176,28 @@ def update_pvs(self):
         xpm_pv = self._config['main']['xpm_pv']
         on_time_idx = self._engine1 * 4
         off_time_idx = (self._engine1 * 4) + 1
+        all_shots_idx = (self._engine1 * 4) + 2
```

Comment:
Why all the index matching here? What does the engine PV look like?

Reply:
The PV is an EPICS 7 normative type PV. It's an NTTable, so it has a structure that's suited towards indexing things out. 
![image](https://github.com/user-attachments/assets/340e0ab7-57a3-4c51-a459-c4b6a86c658d)

Reply:
Ah! I see, that is interesting. I haven't encountered that in a PV yet. I'll add this to my to-do on investigating.